tool/tctl/sso/tester/format_test.go

```diff
@@ -85,6 +85,7 @@ allow:
 deny: {}
 options:
   cert_format: ""
+  create_desktop_user: null
```

Comment:
Can we add an `omitempty` so this doesn't happen? It's odd for a boolean to also be able to be null.

Reply:
All our booleans are nullable so it follows the convention here